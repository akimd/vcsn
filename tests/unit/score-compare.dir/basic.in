 0.12s: a.is_proper()        # a = "", 200000x
 0.27s: b.format("text")     # b = [abc] -> B, 100000x
