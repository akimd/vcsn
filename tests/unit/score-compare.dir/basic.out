 0.12[0m  0.12[0m a.is_proper()        # a = "", 200000x
 0.27[0m  0.27[0m b.format("text")     # b = [abc] -> B, 100000x
